lib/inspec/cli.rb

```diff
@@ -293,6 +293,14 @@ def run_command(opts)
 end
 
 begin
+  # Handle help commands
+  # This allows you to use any of the normal help commands after the normal args.
+  help_commands = ['-h', '--help', 'help']
+  (help_commands & ARGV).each do |cmd|
+    match = ARGV.delete(cmd)
+    ARGV.size > 1 ? ARGV.insert(-2, match) : ARGV.unshift(match)
```

Comment:
The Perl programmer in me is doing a cartwheel, but I do think a comment explaining what you (think) you are doing would be good.

A bare ternary if without assignment.  Sheesh.  I can't believe RuboCop doesn't grouse about that :-)